airbyte-integrations/connectors/source-datadog/acceptance-test-config.yml

```diff
@@ -5,6 +5,8 @@ acceptance_tests:
   spec:
     tests:
       - spec_path: "source_datadog/spec.yaml"
```

Comment:
You removed the spec file, so the spec path here should lead to the manifest, I think. /cc @ChristoGrab